recipe/meta.yaml

```diff
@@ -118,7 +113,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
-        - aiohttp
+        - aiohttp ==3.7
```

Comment:
is there a reason for this strict pinning? might be useful to do a `=3.7` (this would read as "allow `3.7.*`")

in conda, `=` and `==` are treated differently, the latter is more strict

Reply:
aiohttp 3.8.0 has backward incompatible change: https://github.com/ray-project/ray/pull/20261

Reply:
Changed to `=`.

Reply:
That is understandable, but a less strict pinning `=3.7` would also prevent 3.8 (or maybe pin to `<3.8` if e.g. 3.6 also works).

In other words, I would like pinnings to be as flexible as possible.